src/litserve/docker_builder.py

```diff
@@ -67,7 +67,7 @@ def color(text, color_code, action_code=None):
 """
 
 CUDA_DOCKER_TEMPLATE = """# Change CUDA and cuDNN version here
-FROM nvidia/cuda:12.4.1-cudnn-runtime-ubuntu20.04
```

Comment:
looks good but let's use a newer CUDA version. Also @lantiga @ali-alshaar7 do we need cudnn-runtime in docker env?